test/WebJobs.Extensions.Tests/Extensions/CosmosDB/CosmosDBSqlResolutionPolicyTests.cs

```diff
@@ -52,5 +52,30 @@ public void TemplateBind_DuplicateParameters()
             Assert.Single(resolvedAttribute.SqlQueryParameters, p => p.Name == "@foo" && p.Value.ToString() == "1234");
             Assert.Equal("SELECT * FROM c WHERE c.id = @foo AND c.value = @foo", result);
         }
+
+        [Fact]
+        public void TemplateBind_NestedParameter()
```

Comment:
I'd like to see a test that includes both a nested and a non-nested parameter in the same SqlQuery. You should be able to add something here to check that as well.

Reply:
Done